test/functional/feature_fedpeg.py

```diff
@@ -524,6 +531,63 @@ def run_test(self):
         sidechain.generatetoaddress(1, sidechain.getnewaddress())
         assert_equal(sidechain.gettransaction(claim_txid)["confirmations"], 1)
 
+        # Test a confidential pegin.
+        print("Performing a confidential pegin.")
+        # first let sidechain receive some blinded coins
+        blind_addr = sidechain.getnewaddress("", "blech32")
+        sidechain.sendtoaddress(blind_addr, 15)
+        sidechain.generate(5)
+        # start pegin
+        pegin_addrs = sidechain.getpeginaddress()
+        assert_equal(sidechain.decodescript(pegin_addrs["claim_script"])["type"], "witness_v0_keyhash")
+        pegin_addr = addrs["mainchain_address"]
+        txid_fund = parent.sendtoaddress(pegin_addr, 10)
+        # 10+2 confirms required to get into mempool and confirm
+        parent.generate(11)
+        time.sleep(2)
```

Comment:
what is sleep for?

Reply:
no idea :D it was done after the parent send in the rest of the test as well. I think it'll work without here, not sure Travis :D

Reply:
removed it and pushed, let's see :) it works locally